renormalizer/mps/mps.py

```diff
@@ -1597,7 +1624,7 @@ def calc_ft(self):
             dot = self.bra_mps.conj().dot(self.ket_mps)
         else:
             dot = self.bra_mps.conj().expectation(self.mpo, self.ket_mps)
-        return (
+        return complex(
```

Comment:
I don't fully understand why we should transfer the numpy array to python float/complex explicitly?
I found such a transformation in other places.

Reply:
There are two reasons for the transformation.
- The first is related to the JSON output. Numpy float can not be serialized by JSON and has to be converted to Python float first.
- The second is that if Cupy array is involved there could be unexpected behavior. For example
    ```
    a = cp.random.rand(1, 5)
    b = cp.random.rand(5, 1)
    # suppose we calculate the expectation value. Note type of `c` is Cupy float
    c = (a @ b).ravel()[0] 
    d = np.array([c, c]) # `d` become an array of array, not an array of number.
    ```
Converting results that are expected to be numbers to Python scalars can help solving the problem.